tests/src/test/scala/whisk/core/controller/test/ActionsApiTests.scala

```diff
@@ -552,13 +552,6 @@ class ActionsApiTests extends ControllerTestCommon with WhiskActionsApi {
             val response = responseAs[JsObject]
             response.fields("activationId") should not be None
         }
-
-        // it should "ignore &result when invoking nonblocking action"
-        Post(s"$collectionPath/${action.name}?result=true", args) ~> sealRoute(routes(creds)) ~> check {
-            status should be(Accepted)
-            val response = responseAs[JsObject]
-            response.fields("activationId") should not be None
```

Comment:
And per @dubeejw's observation below, this test restored.
This is purely a CLI change at this point. Otherwise you'll need to make a change to the Actions API.